src/site/xdoc/downloads.xml

```diff
@@ -45,24 +45,24 @@ under the License.
     </tr>
     <tr>
       <td style="test-align: left">
-        2.2.4
+        2.2.5
       </td>
       <td style="test-align: left">
-        2020/03/11
+        2020/05/21
       </td>
       <td style="test-align: left">
-        <a href="https://downloads.apache.org/hbase/2.2.4/api_compare_2.2.4RC0_to_2.2.3.html">2.2.4 vs 2.2.3</a>
+        <a href="https://apache.org/hbase/2.2.5/api_compare_2.2.5RC0_to_2.2.4.html">2.2.5 vs 2.2.4</a>
```

Comment:
This link doesn't seem to exist.

Reply:
Yes. This may need some time to work.

Reply:
Ok, let me check it later again.

Reply:
I just checked it again. I think https://downloads.apache.org/hbase/2.2.5/api_compare_2.2.5RC0_to_2.2.4.html is the correct link.

Reply:
My mistake... It exist now.